HUD/GetPauseMenuItemUniqueIdMouseHovered.md

```diff
@@ -0,0 +1,14 @@
+---
+ns: HUD
+aliases: ["0x13C4B962653A5280"]
+---
+## _GET_PAUSE_MENU_ITEM_UNIQUEID_MOUSE_HOVERED
+
+```c
+// 0x13C4B962653A5280 
+void _GET_PAUSE_MENU_ITEM_UNIQUEID_MOUSE_HOVERED(
```

Comment:
It doesn't return anything anymore because you changed it's type to a void. You also broke the end of this line by removing the closing `)` followed by `;`.